### ISSUE TITLE
 Some thoughts on the overview.

### ISSUE THREAD

**Issue by Stevengre** [runtimeverification/gitbook-kontrol#58]
Here, I want to express some of my comments about current `Overview` section. If @palinatolmach agree with me, I'll try to fix them one by one.

---

1. The image appears to have a border, which is not quite perfect? And I'm wondering if it would be better to replace the title in the top left corner with this image.
2. Another question is about the slogan. Could we make it more attractive using the word `kontrol`? I don't know why we use the word `kontrol` to call our tool. But maybe: `Kontrol your smart contract behavior with EFFORTLESS FORMAL VERIFICATION`? or use `EASY` rather than `EFFORTLESS`?

![image](https://github.com/user-attachments/assets/3a1ffe75-949e-4c94-ac90-603a42ab2975)

---

3. Regarding this first page, I want to make it more attractive with fewer technical stuffs that hard to follow. In fact, it's me that hard to get the key point. The following is what I want to replace these three

**Comment by palinatolmach** [comment]
@Stevengre thanks for the suggestions! 
1. Fully agree about the borders in the logo image, not sure about the top left corner — maybe we want to keep it for indexing purposes and such? 
2. Ageed; on the website, we now use "Take Kontrol of Smart Contracts" and then "Formal Verification made simple, so you don't have to settle for fuzzing", so perhaps we should use it here too: https://kontrol.runtimeverification.com/.
3. I think the end of the sentence got cut off, can you please check?

**Comment by Stevengre** [comment]
Thank you for your quick response. I write the last sentence and think maybe I can just commit a PR for better discussion.

**Comment by Stevengre** [comment]
About the logo, If we want to keep it, is there any ways to remove the borders? I think it might because some settings in the gitbook causing this strange rendering.

**Comment by Stevengre** [comment]
Hi @palinatolmach! I got this error when I want to publish a new branch.

```
remote: Permission to runtimeverification/gitbook-kontrol.git denied to Stevengre.
```

Should I folk another repo? Or could you please grant me the right?

**Comment by Stevengre** [comment]
Related PR: #59
